dateutil/parser/_parser.py

```diff
@@ -324,19 +324,17 @@ def jump(self, name):
         return name.lower() in self._jump
 
     def weekday(self, name):
-        if len(name) >= min(len(n) for n in self._weekdays.keys()):
```

Comment:
This was recently added [here](https://github.com/dateutil/dateutil/commit/505b9b22605d05fa5c6d41a66ea95f88022018fd#diff-ed798069162e7ab7db9a45da6c4a076fL312), replacing a `len(name) >= 3` check that was [part of the original commit in this repo](https://github.com/dateutil/dateutil/commit/68ae2757ae15c84bf947d47a82a314b3b975bc9b).

I think it was just there for performance reasons.